com.super_productivity.SuperProductivity.yml

```diff
@@ -0,0 +1,73 @@
+app-id: com.super_productivity.SuperProductivity
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-icon: superproductivity
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  # Use socket=x11 instead of socket=fallback-x11 to make it easier for people to just
+  # unset ELECTRON_OZONE_PLATFORM_HINT if they have problems with Electron's
+  # experimental Wayland support
```

Comment:
```suggestion
```

either is work or it doesn't. there is no middle ground where people have to tweak things.